new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in feature1 branch 
```

Comment:
It's good that you decide to add this line of code

Reply:
Thanks for the comment